src/main/java/com/openshift/internal/restclient/model/user/OpenShiftUser.java

```diff
@@ -37,4 +39,13 @@ public String getUID() {
         return asString("metadata.uid");
```

Comment:
you might consider adding this identifier to ```com.openshift.internal.restclient.model.properties.ResourcePropertyKeys```

Reply:
Good Idea. Fixed.